driver/modern_bpf/helpers/store/auxmap_store_params.h

```diff
@@ -787,18 +787,24 @@ static __always_inline void auxmap__store_socktuple_param(struct auxiliary_map *
 		 */
 		push__u8(auxmap->data, &auxmap->payload_pos, socket_family_to_scap(socket_family));
 		if(direction == OUTBOUND) {
-			push__u64(auxmap->data, &auxmap->payload_pos, (uint64_t)socket_remote);
+			push__u64(auxmap->data, &auxmap->payload_pos, (uint64_t)socket_peer);
 			push__u64(auxmap->data, &auxmap->payload_pos, (uint64_t)socket_local);
-			path = BPF_CORE_READ(socket_remote, addr, name[0].sun_path);
+			struct sockaddr_un usrsockaddr_un = {};
```

Comment:
uhm here we are copying the struct into the ebpf stack but `path = usrsockaddr_un.sun_path` will point to nothing when we exit the scope... probably we need to move it outside this if `if(direction == OUTBOUND) {`

Reply:
Great catch! Thanks!